kale/loaddata/video_access.py

```diff
@@ -188,31 +227,34 @@ def get_test(self):
             annotationfile_path=self._test_list,
             num_segments=1,
             frames_per_segment=self._frames_per_segment,
-            imagefile_template='frame_{:010d}.jpg',
-            transform=self._transform['test'],
+            imagefile_template="frame_{:010d}.jpg",
+            transform=self._transform["test"],
             random_shift=False,
             test_mode=True,
             image_modality=self._image_modality,
-            dataset_split='test',
+            dataset_split="test",
             n_classes=self._n_classes,
         )
 
 
 class GTEADatasetAccess(VideoDatasetAccess):
     """GTEA data loader"""
+
     def get_train(self):
         return BasicVideoDataset(
             root_path=self._data_path,
             annotationfile_path=self._train_list,
             num_segments=1,
             frames_per_segment=self._frames_per_segment,
-            imagefile_template='frame_{:010d}.jpg' if self._image_modality in ['rgb'] else 'flow_{}_{:010d}.jpg',
-            transform=self._transform['train'],
+            imagefile_template="frame_{:010d}.jpg"
+            if self._image_modality in ["rgb"]
+            else "flow_{}_{:010d}.jpg",
```

Comment:
It looks good, but the previous one may be more clear to read. Just my opinion.

Reply:
Many thanks for pointing it out Xianyuan. I figured out the problem (via the playground). I did not configure black to 120  when running it. This is bad. I may need to redo these commits in a new branch rather than merging this one.